reference/entry.json

```diff
@@ -2006,6 +2006,259 @@
           "Places"
         ],
         "x-internal": false
+      },
+      "put": {
+        "summary": "",
+        "operationId": "put-places",
+        "responses": {
+          "200": {
+            "description": "The place was updated successfully",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "example": "c8e6ff82-8b30-4295-b937-ab2f4f6ab4bf",
+                      "format": "uuid",
+                      "description": "The id of the updated place."
+                    },
+                    "placeId": {
+                      "type": "string",
+                      "deprecated": true,
+                      "example": "c8e6ff82-8b30-4295-b937-ab2f4f6ab4bf",
+                      "format": "uuid",
+                      "description": "The id of the updated place (deprecated and replaced with `id`)."
+                    },
+                    "url": {
+                      "type": "string",
+                      "example": "https://io.uitdatabank.be/places/c8e6ff82-8b30-4295-b937-ab2f4f6ab4bf",
+                      "format": "uri",
+                      "description": "The url of the JSON-LD representation of the updated place."
+                    }
+                  },
+                  "required": [
+                    "id",
+                    "placeId",
+                    "url"
+                  ]
```

Comment:
I think the response will also include a `commandId`

Reply:
Although I'd like to make the imports work synchronously so it might be removed in the future.